README.md

```diff
@@ -41,7 +41,30 @@ const helpers = {
 amphoraHtml.addHelpers(helpers);
 ```
 
-Now that you have registered your helpers and provided a root path which Amphora HTML can work from, you can register your renderer with Amphora. Registering consists of providing a `renderers` object whose keys are the extension of an HTTP request and whose values are the renderer. You can also specify a `default` property whose value is the extension that Amphora should default to when rendering. This is handy for rendering routes who don't end in extensions, such as `mycoolsite.com/about/`.
+You can add amphora-html plugins like this:
```

Comment:
Picky but would like a more expanded explanation of why you would want to use a plugin. Should define it's limitations that we've discussed (no css/js trimming). Would definitely put a word of warning in to say not to go overboard.

Reply:
Oh, and say that the plugin functionality is looking to be expanded and will include more in the future. If more functionality should be added, ask people to file an issue